UnitTests/GitUITests/UserControls/InteractiveGitActionControlTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using GitUI.UserControls;
+using NUnit.Framework;
+
+namespace GitUITests.UserControls
+{
+    [Apartment(ApartmentState.STA)]
+    public class InteractiveGitActionControlTests
+    {
+        private InteractiveGitActionControl.TestAccessor _accessor;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _accessor = new InteractiveGitActionControl().GetTestAccessor();
+        }
+
+        [TestCase(InteractiveGitActionControl.State.Rebase, false)]
+        [TestCase(InteractiveGitActionControl.State.Rebase, true)]
+        [TestCase(InteractiveGitActionControl.State.Merge, false)]
+        [TestCase(InteractiveGitActionControl.State.Merge, true)]
+        [TestCase(InteractiveGitActionControl.State.Patch, false)]
+        [TestCase(InteractiveGitActionControl.State.Patch, true)]
+        [TestCase(InteractiveGitActionControl.State.Bisect, false)]
+        [TestCase(InteractiveGitActionControl.State.None, false)]
+        public void SetState(InteractiveGitActionControl.State state, bool conflicts)
+        {
+            _accessor.SetState(state, conflicts);
+
+            Assert.AreEqual(state, _accessor.State);
+            Assert.AreEqual(conflicts, _accessor.HasConflicts);
+        }
```

Comment:
We need to add other tests to verify that:
- the control is shown/hidden
- buttons added or not
- a specific button is visible for a specific action